docs/core/tools/dotnet-workload-sets.md

```diff
@@ -0,0 +1,167 @@
+---
+title: .NET SDK workload sets
+description: "Learn how to use workload sets to control which versions of workloads get installed, updated, or restored."
+author: tdykstra
+ms.author: tdykstra
+ms.service: dotnet
+ms.topic: how-to
+ms.date: 08/02/2024
+
+#customer intent: As a developer using optional workloads, I want to keep my workload versions in sync so that I can avoid conflicts betwween versions. I also want to avoid unexpected workload updates, so that I can avoid disruptions in development.
+---
+<!--https://aka.ms/patterns-feedback-->
+
+# .NET SDK workload sets
+
+The *workload sets* feature provides a version number that represents a group of .NET SDK workloads. The [install](dotnet-workload-install.md), [update](dotnet-workload-update.md), and [restore](dotnet-workload-restore.md) commands use this number in *workload-set update mode* to provide the following benefits:
+
+* You control the cadence of change for installed workload versions. The alternative mode of operation without using workload sets is called *loose manifests update mode*. In this mode, workloads are updated automatically as new versions of individual workloads are released onto any configured NuGet feeds. In workload set update mode, workloads stay at a specific workload set version until you explicitly change that version.
+* You can install and update a combination of workload versions that ship at the same time and are known to work together.
+* You can ensure that everyone on your team is always working on the same workload versions.
+* You don't have to use a rollback file to specify what workload version you want to be on.
+
+Here are some ways you can use workload sets:
+
+* "Pin" the `install` command to a specific workload-set version.
+* Update installed workloads to the latest available workload-set version.
+* Update to a specified workload-set version.
+* Specify the workload set version in global.json.
+* Check your current update mode and workload-set version.
+
+And you can still choose to install, update, or restore to the latest version of each individual workload, ignoring workload sets.
+
+## Prerequisites
+
+* [.NET 8.0.400 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) or later.
+
+  In 8.0.400 SDK, the `dotnet workload` commands are in workload-set update mode only when it's explicitly selected. Starting in [.NET 9 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/9.0), `dotnet workload` commands are in workload-set update mode by default.
```

Comment:
I don't think we can say that .NET 9 is opt-out for certain, so I think we should just say what we are certain of here: .NET 8.0.400 is opt-in.